docs/openapi/definitions/auth.yaml

```diff
@@ -0,0 +1,281 @@
+  /refresh:
+    get:
+      tags:
+        - provider
+      summary: Handles refreshing of tokens due to reload or expiry
+      description:
+        If supported by the provider, the `/refresh` endpoint is responsible for
+        refreshing the tokens using a refresh-token stored in a cookie
+      parameters:
+        - name: scope
+          in: query
+          description: Scope requested for the auth session
+          required: true
+          schema:
+            type: string
+            example: email
+        - name: optional
+          in: query
+          description: Prevent the popup from being displayed
+          schema:
+            type: string
+        - name: x-requested-with
+          in: header
+          description: X-Requested-With header preventing CSRF
+          required: true
+          schema:
+            type: string
+            example: 'x-requested-with: XMLHttpRequest'
+        - name: provider-refresh-token
+          in: cookie
+          description: Refresh token used to refresh the current session
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Successfully refreshed the session
+        401:
+          description: Validation error
+  /logout:
+    post:
+      tags:
+        - provider
+      summary: Logs user out of the current provider
+      description:
+        When logging out the current session is removed and if present the
+        cookie containing the refresh token is also removed
+      parameters:
+        - name: x-requested-with
+          in: header
+          description: X-Requested-With header preventing CSRF
+          required: true
+          schema:
+            type: string
+            example: 'x-requested-with: XMLHttpRequest'
+      responses:
+        200:
+          description: Successfully logged out
+          headers:
+            Set-Cookie:
+              description: Refresh cookie
+              schema:
+                $ref: '#/components/schemas/Refresh-Cookie'
+        401:
+          description: Validation error
+components:
+  schemas:
+    Nonce-Cookie:
+      type: object
+      properties:
+        provider-nonce:
+          type: string
+        maxAge:
+          type: integer
+        secure:
+          type: boolean
+        sameSite:
+          type: string
+        domain:
+          type: string
+        path:
+          type: string
+        httpOnly:
+          type: boolean
+    Refresh-Cookie:
+      type: object
+      properties:
+        provider-refresh-token:
+          type: string
+        maxAge:
+          type: integer
+        secure:
+          type: boolean
+        sameSite:
+          type: string
+        domain:
+          type: string
+        path:
+          type: string
+        httpOnly:
+          type: boolean
+    AuthResponse:
+      type: object
+      properties:
+        type:
+          type: string
```

Comment:
is it possible to define an actual exact value for this here? since it's supposed to be `"auth-response"` and nothing else.

same below